src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1947,6 +1993,17 @@
                     }
                 },
 
+                "translation": {
+                    "description": "Describes the translations for the analysis tool.",
```

Comment:
>. [](start = 84, length = 1)

What do you think of this: "Describes the available translations of the messages in the log file."

Reply:
like it very much!

---
In reply to: [263057306](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263057306) [](ancestors = 263057306)

Reply:
I don't understand this, the 'translations' array doesn't describe the available translations for the analysis tool. Instead, the array itself comprises a set of translations of the localized data provided by the tool.

---
In reply to: [263164891](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263164891) [](ancestors = 263164891,263057306)

Reply:
Based on Larry's other comments, this should be something like so (larry please tune)

"Provides localized strings, for one or more languages, for the current run.

---
In reply to: [263169512](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263169512) [](ancestors = 263169512,263164891,263057306)

Reply:
Responding to @michaelcfanning (who I think is commenting in the GitHub web UI, so his comments aren't threading properly): Yes, that's good: "The set of available translations of the localized data provided by the tool."

---
In reply to: [263057306](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263057306) [](ancestors = 263057306)

Reply:
thanks both! 

---
In reply to: [263173538](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263173538) [](ancestors = 263173538,263057306)